examples/get-started/README.md

```diff
@@ -0,0 +1,151 @@
+# Get started with Aiven Provider for Terraform 
+
+Introduction
+
+
+ 
+
+## Prerequisites
+
+* [Install Terraform](https://www.terraform.io/downloads)
+* [Sign up for Aiven](https://console.aiven.io/signup?utm_source=github&utm_medium=organic&utm_campaign=devportal&utm_content=repo)
+* [Create an authentication token](https://docs.aiven.io/docs/platform/howto/create_authentication_token.html)
+
+## Create your first Aiven resources
+
+1. Ensure that you have Terraform v0.13.0 or higher installed. To check the version, run:  
+
+```sh
+$ terraform --version 
+```
+
+The output is similar to the following:
+
+```sh
+Terraform v1.6.2
++ provider registry.terraform.io/aiven/aiven v4.9.2
+```
+
+2. Clone this repository.
```

Comment:
cloning the repository is only needed for development purposes, for regular usage of the Terraform provider it's not needed

Reply:
This step was added here and in other examples in anticipation of creating a dedicated examples repo. That way everything can just be moved without editing the steps - does that work or is it better to still generalize the instructions?

Reply:
Ah, I see the point now. We should definitely consider a dedicated examples repo in future. For now, let's get forward with it.